src/foobar/admin.py

```diff
@@ -192,6 +192,25 @@ def changelist_view(self, request, extra_context=None):
         return response
 
 
+@admin.register(models.WalletLogEntry)
```

Comment:
Everything here should be read-only. Also, nobody should be able to delete log entries.
![image](https://cloud.githubusercontent.com/assets/74944/23128105/ea0f982e-f77d-11e6-8e24-ef22eb7f53d5.png)